huxley/www/js/components/RegistrationView.js

```diff
@@ -90,12 +92,20 @@ var RegistrationView = React.createClass({
   },
 
   componentDidMount: function() {
-    CountryStore.getCountries(function(countries) {
-      this.setState({countries: countries});
-    }.bind(this));
-    CommitteeStore.getSpecialCommittees(function(committees) {
+    this._committeesToken = CommitteeStore.addListener(() => {
+      var committees = CommitteeStore.getSpecialCommittees();
```

Comment:
not gonna comment on every instance of this, but these don't need to be variables